p4-16/spec/P4-16-spec.mdk

```diff
@@ -2287,8 +2287,9 @@ one bit long field. This declaration introduces a new identifier in the
 current scope; the type can be referred to using this identifier. A header is
 similar to a `struct` in C, containing all the specified fields. However, in
 addition, a header also contains a hidden Boolean "validity" field. When the
-"validity" bit is `true` we say that the "header is valid". When a header is
-created its "validity" bit is automatically set to `false`. The "validity"
+"validity" bit is `true` we say that the "header is valid". When a local
+variable with a header type is
+declared, its "validity" bit is automatically set to `false`. The "validity"
```

Comment:
This mention of the initialization requirement on a value with type header already seemed pretty explicit to me.  The only change here is to change the phrase "when a header is created" to "when a local variable with a header type is declared", since the proposed replacement seems a little clearer to me than "created".